go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/bestchains/bestchains-contracts
 
-go 1.18
+go 1.20
```

Comment:
1. update golang image version in dockerfile
2. push this new golang image to private registry
3. test chaincode build